lightning/src/routing/scoring.rs

```diff
@@ -476,6 +476,9 @@ where L::Target: Logger {
 	network_graph: G,
 	logger: L,
 	channel_liquidities: ChannelLiquidities,
+	/// The last time we were given via a [`ScoreUpdate`] method. This does not imply that we've
+	/// decayed every liquidity bound up to that time.
+	last_duration_since_epoch: Duration,
```

Comment:
The way time is used, the "since epoch" may be unnecessary? Whereas adding an indication as to what time this is could be useful. Simply `last_update_time`?